api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/subscriptions_controller.rb

```diff
@@ -128,9 +109,19 @@ def fetch_subscriptions(organization_id)
 
   def fetch_subscription(organization_id, id, includes = nil)
     rel = MnoEnterprise::Subscription
+            .apply_query_params(params)
             .with_params(_metadata: { act_as_manager: current_user.id })
             .where(organization_id: organization_id, id: id)
     rel = rel.includes(*includes) if includes.present?
     rel.first
   end
+
+  def set_staged_subscription_params
+    params[:where] ||= {}
+    params[:where][:subscription_status_in] = cart_subscription_param.present? ? 'staged' : 'non_staged'
```

Comment:
Could we reuse the backend semantic and have the filters `staged` / `visible` ?
https://github.com/maestrano/maestrano-hub/pull/1004/files